rspec/spec_commands/spring.py

```diff
@@ -12,5 +12,4 @@ def spring_included(self):
     gemfile_path = self.context.gemfile_path()
     if not gemfile_path: return
 
-    return open(gemfile_path, 'r').read().find('spring') > 0
-
+    return open(gemfile_path, "r").read().find("spring-commands-rspec") > 0
```

Comment:
Checking for both `spring` and `spring-commands-rspec` substrings is redundant, so I'm checking only for the latter.